### DISCUSSION TITLE
Logo Suggestion

### DISCUSSION THREAD

**Discussion by ahangarha** [alpinejs/alpine#1281] [3. Feature ideas]
I have tried to make a logo for Alpine. It is inspired by Vue logo (for obvious reason).

![image](https://user-images.githubusercontent.com/11241315/116083437-d81af680-a6b1-11eb-8ed0-207e0cc1e94e.png)

**Comment by lovedder1995** [comment]
V for Vue and, A for Alpine?

![alpine](https://user-images.githubusercontent.com/29613190/116098442-b2b9c880-a670-11eb-98f7-6631100b2354.jpg)
